client/src/leap/soledad/client/api.py

```diff
@@ -656,6 +659,20 @@ def sync(self, defer_decryption=True):
             defer_decryption=defer_decryption)
 
         def _sync_callback(local_gen):
+            self._last_received_docs = docs = self._dbsyncer.received_docs
+
+            # Post-Sync Hooks
+            synced_plugin = soledad_interfaces.ISoledadPostSyncPlugin
```

Comment:
Maybe preciosism, but i'd change the name from `synced_plugin` to `iface`, and maybe move this line to inside the if below.